plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -712,39 +762,31 @@ private String getLongInClause(int start, int length)
     }
 
     private AutoCloseable withSchema(String schema)
-            throws Exception
     {
         execute(format("CREATE SCHEMA %s", schema));
-        return () -> {
-            try {
-                execute(format("DROP SCHEMA %s", schema));
-            }
-            catch (SQLException e) {
-                throw new RuntimeException(e);
-            }
-        };
+        return () -> execute("DROP SCHEMA %s" + schema);
```

Comment:
You probably meant to use `String#format`, but I guess you already know that (from the failing test)